src/components/forms/Combobox.tsx

```diff
@@ -883,6 +883,26 @@ export function ComboboxAsync<T = any>({
 
         if (inputOptions.value && inputOptions.value.toString().length > 0) {
           currentUrl.searchParams.set('with', inputOptions.value.toString());
+
+          if (currentUrl.searchParams.get('sort')) {
+            currentUrl.searchParams.delete('sort');
+          }
```

Comment:
In the moment when the 'with' parameter is set, we need to ensure the removal of the 'sort' parameter if it exists.